README.md

```diff
@@ -187,15 +237,20 @@ ecosystem.
 
 <!-- TODO(portable-rustls) CLEANUP THIS NOTE & IMPROVE THE INFO HERE: -->
 HIGHLY RECOMMENDED TO LOOK INTO `provider-example` SUBDIRECTORY ([`provider-example`](provider-example/)) IN THIS FORK; SEE ESPECIALLY `provider-example/Cargo.toml`.
-AS DISCUSSED ABOVE, HIGHLY RECOMMENDED TO PUT DEPENDENCY LIKE THIS INTO `Cargo.toml`: `rustls = { package = "portable-rustls" ... }`
+
+As described above, it is (highly) recommended to add dependency on this fork as follows in `Cargo.toml`:
+
+>```rust,ignore
+>rustls = { package = "portable-rustls", features=[...], ... }
+>```
 
 See the [Making a custom CryptoProvider] section of the documentation for more information
 on this topic.
 
-<!-- TODO(portable-rustls) UPDATE WITH BETTER REFERENCES FOR THIS FORK: -->
-[`custom-provider`]: https://github.com/rustls/rustls/tree/main/provider-example/
+<!-- TODO REFERENCE MAIN DEV BRANCH INSTEAD OF OLD TAG ON GITHUB -->
+[`custom-provider`]: https://https://github.com/brody4hire/portable-rustls/tree/portable-rustls-0.0.1/provider-example/
 [`RustCrypto`]: https://github.com/RustCrypto
```

Comment:
I definitely want to fix this before merging - blocked by issue described in: #43